Testscripts/Linux/build_test_vpp.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+# This script will build and test VPP.
+
+HOMEDIR=$(pwd)
+export VPP_DIR="${HOMEDIR}/vpp"
+export RTE_SDK="${HOMEDIR}/dpdk"
+export RTE_TARGET="x86_64-native-linuxapp-gcc"
+UTIL_FILE="./utils.sh"
+
+# Source utils.sh
+. utils.sh || {
+	echo "ERROR: unable to source utils.sh!"
+	echo "TestAborted" > state.txt
+	exit 0
+}
+
+# Source constants file and initialize most common variables
+UtilsInit
+
+function build_test_vpp () {
+	SetTestStateRunning
+	LogMsg "Configuring ${1} ${DISTRO_NAME} ${DISTRO_VERSION} for VPP test..."
+	packages=(git)
+	package_manager=""
+	package_manager_install_flags=""
+	package_type=""
+	case "${DISTRO_NAME}" in
+		oracle|rhel|centos)
```

Comment:
In PS, it filters out others except Redhat and Ubuntu. This line should be only 'rhel'. The line 46 is 'ubuntu'.

Reply:
the PS rules do not apply, as it s bash.